src/bundle/Resources/views/content/tabs/locations.html.twig

```diff
@@ -18,7 +22,7 @@
     {% if locations is defined %}
         {% for location in locations %}
         <tr>
-            <td></td>
+            <td>{{ form_widget(actionsForm.removeLocations[location.id], {'attr': {'disabled': location.main}}) }}</td>
```

Comment:
besides disabling the checkbox when it's the main Location, it should also be disabled if the current user does not have the right to delete the Location. I'm commenting in the template but I don't know where this should be handled